og_migrate/src/Plugin/migrate/destination/OgField.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\og_migrate\Plugin\migrate\destination;
+
+use Drupal\migrate\Plugin\migrate\destination\DestinationBase;
+use Drupal\migrate\Plugin\MigrationInterface;
+use Drupal\migrate\Row;
+use Drupal\og\Og;
+
+/**
+ * Creates og_audience field on entity bundle.
+ *
+ * @MigrateDestination(
+ *   id = "og_field"
+ * )
+ *
+ * @internal
+ */
+class OgField extends DestinationBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function import(Row $row, array $old_destination_id_values = []) {
+    try {
+      $field = Og::createField(
+        $row->getDestinationProperty('plugin_id'),
+        $row->getDestinationProperty('entity_type'),
+        $row->getDestinationProperty('entity_bundle')
+        // @todo worth it to migrate field settings?
```

Comment:
@mradcliffe, I don't think it's worth it but instead of adding a todo here perhaps we could output a message to the user?

Reply:
Thanks, that makes sense.